source/educators/how-tos/drag_and_drop_deprecated.rst

```diff
@@ -39,9 +39,9 @@ objects to a specific location on an image.
 .. image:: /_images/educator_how_tos/DragAndDropProblem.png
  :alt: Image of a drag and drop problem.
 
-*********************************
+******************************
 Adding a Drag and Drop Problem
```

Comment:
I think we can just remove this page, since the problem type has been deprecated for quite some time.